packages/next-swc/crates/core/src/next_font_loaders/font_functions_collector.rs

```diff
@@ -0,0 +1,70 @@
+use swc_core::common::errors::HANDLER;
+use swc_core::ecma::ast::*;
+use swc_core::ecma::visit::noop_visit_type;
+use swc_core::ecma::visit::Visit;
+
+pub struct FontFunctionsCollector<'a> {
+    pub font_loaders: &'a [String],
+    pub state: &'a mut super::State,
+}
+
+impl<'a> Visit for FontFunctionsCollector<'a> {
+    noop_visit_type!();
+
+    fn visit_import_decl(&mut self, import_decl: &ImportDecl) {
+        if self
+            .font_loaders
+            .contains(&String::from(&*import_decl.src.value))
```

Comment:
`String::from(&*import_decl.src.value)` allocates string just to check if a string is in a list.